style.css

```diff
@@ -168,12 +169,14 @@ image {
   font-weight: 600;
 }
 
-.frame-img {
+.frame-img-1,
+.frame-img-2 {
   color: var(--font-fifth-color);
   font-weight: 600;
```

Comment:
### Optional
- [ ] It would be better if we add the project screenshot to our README file, this gives the first impression of the project build and helps in making our repository professional